### DISCUSSION TITLE
"records match" clarification in maths

### DISCUSSION THREAD

**Discussion by rlshuhart** [moj-analytical-services/splink#819] [Q&A] [answered]
Hi,

First, thanks to Robin, all the others, and the funders for creating or enabling the creation of splink. I think this will be tremendously helpful for my use case of matching business entities. And thank you, Robin, for creating the training materials on probabilistic linkage.

I'm hoping for a little assistance in understanding the maths in the training material. I'm attempting to calculate the probabilities manually to understand this better, but having a hard time understanding what is a matched record or "records match" as given in the probability statements at https://www.robinlinacre.com/maths_of_fellegi_sunter/. 

**Pr(first name does not match∣record match)** - The "record match" is throwing me off. How can a record match if the first name does not match? The definition of "record match" is unclear to me. Is this considering fuzzy thresholds? So a row matches because similarity thresholds are met, but the exact match is False?

**Comment by RobinL** [comment, marked as answer]
The definition of 'records match' is 'the underlying entity is the same' i.e. it's the same person.

The idea is that sometimes typos, omissions or other errors cause the data to be different despite the underlying entity being the same

For example, consider the following records:

Rob Smith, postcode AB1 2CD, dob 2010-09-01
Robin Smith, postcode AB1 2CD, dob 2010-09-01

These are almost certainly the same person, despite the first name not matching - and this is the probability the model is trying to quantify

**Reply by rlshuhart** [reply]
Thanks Robin! I see I was losing sight of the theory. Very helpful.
